File: app/src/main/java/ca/com/toronto/model/Geometry.java
package ca.com.toronto.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Royal_L on 23-Oct-17.
 */

public class Geometry {

    @SerializedName("location")
    @Expose
    private Location location;

    /**
     *
     * @return
     * The location
     */
    public Location getLocation() {
        return location;
    }

    /**
     *
     * @param location
     * The location
     */
    public void setLocation(Location location) {
        this.location = location;
    }
}

File: app/src/main/java/ca/com/toronto/model/Geometry.kt
package ca.com.toronto.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Royal_L on 23-Oct-17.
 */
data class Geometry(
    /**
     * @param [location]
     * @return The location
     */
    @SerializedName("location")
    @Expose
    var location: Location? = null
)
